sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -22,10 +22,14 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "evtimer_msg.h"
 #include "kernel_types.h"
 #include "mutex.h"
 #include "net/eui64.h"
 #include "net/ipv6/addr.h"
+#ifdef MODULE_GNRC_IPV6
```

Comment:
why would you add guards around `ipv6.h`, but not `ipv6/nib/nc.h` and `ipv6/nib/conf.h`? Is it possible to use `nib` without `ipv6`? (: